go.mod

```diff
@@ -76,6 +76,7 @@ require (
 	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
 	golang.org/x/sync v0.0.0-20190423024810-112230192c58
 	golang.org/x/sys v0.0.0-20190616124812-15dcb6c0061f
+	golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543 // indirect
```

Comment:
I'd like to know if this is ok before merging ?

Reply:
Did this get automatically added when you did a `make build`, right?